routes/user.js

```diff
@@ -25,18 +40,33 @@ router.get("/user", authenticateToken, (req, res) => {
 });
 
 router.post("/user/create", async (req, res) => {
-  try {
-    const hashedPassword = await bcrypt.hash(req.body.password, 10);
-    const user = { username: req.body.name, password: hashedPassword };
+  let findUser = userDB.prepare("SELECT * FROM users WHERE username = ?");
 
-    let insert = "INSERT INTO users (username, password) VALUES (?,?)";
-    userDB.run(insert, [user.username, user.password]);
+  //Change to get
+  findUser.each(
+    req.body.name,
+    (error, row) => {},
```

Comment:
This row is not used? Needed?

Reply:
Yeah otherwise the rest of the code is used in the loop and it causes errors